runtime/onert/core/include/util/Exceptions.h

```diff
@@ -19,26 +19,41 @@
 
 #include <string>
 
+#include <ir/DataType.h>
+
 namespace onert
 {
 
-class OnertException : public std::exception
+class Exception : public std::exception
```

Comment:
I'm not sure that the change of the existing name is required here. What was the reason behind it?

Reply:
I also like to remove the redundancy. However, it is beyond this PR. Could you please don't change this in this PR? We may get other's opinion in separate PR.

Reply:
Submitted as a separate PR: https://github.com/Samsung/ONE/pull/16257